djangofromscratch/JWTLogin/managers.py

```diff
@@ -1,43 +1,16 @@
-import re
-
 from django.contrib.auth.hashers import make_password
 from django.contrib.auth.models import UserManager as DjangoUserManager
+from django.contrib.auth.password_validation import validate_password
 
 
 class UserManager(DjangoUserManager):
-    # Define a function for
```

Comment:
I wanted to apply the rule on all email and password field so that the rule gets applied other than only creation of user, like for example, when updating emails , this rule will not get applied. So, I shifted this code of check_email fn to User model's class and in the charfield validator parameter, I passed the fn. For password, django provides it's own validation so I removed the password validation and used that.